src/MerchantCenter/MerchantStatuses.php

```diff
@@ -108,15 +108,18 @@ public function __construct() {
 	public function get_product_statistics( bool $force_refresh = false ): array {
 		$this->maybe_refresh_status_data( $force_refresh );
 
-		$counting_stats = $this->mc_statuses['statistics'];
-		$counting_stats = array_merge(
-			[ 'active' => $counting_stats[ MCStatus::PARTIALLY_APPROVED ] + $counting_stats[ MCStatus::APPROVED ] ],
-			$counting_stats
-		);
-		unset( $counting_stats[ MCStatus::PARTIALLY_APPROVED ], $counting_stats[ MCStatus::APPROVED ] );
+		$counting_stats = isset( $this->mc_statuses['statistics'] ) ? $this->mc_statuses['statistics'] : null;
+
+		if ( is_array( $counting_stats ) ) {
+			$counting_stats = array_merge(
+				[ 'active' => $counting_stats[ MCStatus::PARTIALLY_APPROVED ] + $counting_stats[ MCStatus::APPROVED ] ],
+				$counting_stats
+			);
+			unset( $counting_stats[ MCStatus::PARTIALLY_APPROVED ], $counting_stats[ MCStatus::APPROVED ] );
```

Comment:
❓ Just curious. Why we need to unset this?

Reply:
We are combining partially approved and approved together as active. If we don't unset them then they'll still show up in the results.